blueprints/model/index.js

```diff
@@ -35,26 +39,44 @@ module.exports = {
         var camelizedNamePlural = inflection.pluralize(camelizedName);
         attr = dsAttr(dasherizedForeignModelSingular, dasherizedType);
         attrs.push(camelizedNamePlural + ': ' + attr);
+        shouldImportHasMany = true;
       } else if (/belongs-to/.test(dasherizedType)) {
         attr = dsAttr(dasherizedForeignModel, dasherizedType);
         attrs.push(camelizedName + ': ' + attr);
+        shouldImportBelongsTo = true;
       } else {
         attr = dsAttr(dasherizedName, dasherizedType);
         attrs.push(camelizedName + ': ' + attr);
+        shouldImportAttr = true;
       }
 
       if (/has-many|belongs-to/.test(dasherizedType)) {
         needs.push("'model:" + dasherizedForeignModelSingular + "'");
       }
     }
+
     var needsDeduplicated = needs.filter(function(need, i) {
       return needs.indexOf(need) === i;
     });
 
+    if (shouldImportAttr) {
+      importStatements.push('import attr from \'ember-data/attr\';');
+    }
+
+    if (shouldImportBelongsTo && shouldImportHasMany) {
+      importStatements.push('import { belongsTo, hasMany } from \'ember-data/relationships\';');
```

Comment:
Out of scope for this pr but I've been thinking it may be useful to add a path that allows users to import `belongsTo`, `hasMany` and `attr` all on the same line.